packages/round-manager/src/features/api/votingStrategy/votingStrategy.ts

```diff
@@ -9,22 +12,32 @@ import { qfVotingStrategyFactoryContract } from "../contracts";
  * @param signerOrProvider
  * @returns
  */
-export const deployQFVotingContract = async (
-  signerOrProvider: Signer
+export const deployVotingContract = async (
+  signerOrProvider: Signer,
+  isQF: boolean
 ): Promise<{ votingContractAddress: string }> => {
   try {
     const chainId = await signerOrProvider.getChainId();
 
-    const _QFVotingStrategyFactory = qfVotingStrategyFactoryContract(chainId); //roundFactoryContract(chainId);
-    const qfVotingStrategyFactory = new ethers.Contract(
-      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-      _QFVotingStrategyFactory.address!,
-      _QFVotingStrategyFactory.abi,
+    if (!isQF) {
+      // Note: In direct-rounds, we use a dummy contract for the voting contract.
+      return Promise.resolve({
```

Comment:
Since `deployVotingContract` is `async`, the return value should already be wrapped in a promise, so probably the `Promise.resolve` call here can be skipped.